README.md

```diff
@@ -161,10 +161,12 @@ function getFruitCount() {
 
 > See also [compose](#compose).
 
-You can use `get` when performing asynchronous operations as well:
+### `fetch`
+
+You can use `fetch` when performing asynchronous operations as well:
```

Comment:
I don't think that "as well" is appropriate here since it isn't connected with `get` anymore.